lib/Elastica/JSON.php

```diff
@@ -60,6 +60,9 @@ public static function stringify($args/* inherit from json_encode */)
         // extract arguments
         $args = \func_get_args();
 
+        // set defaults
+        isset($args[1]) ? $args[1] |= JSON_PRESERVE_ZERO_FRACTION : $args[1] = JSON_PRESERVE_ZERO_FRACTION;
```

Comment:
Had to read this line a few times but makes sense now :-)

Reply:
Yeah, kinda cryptic. I couldn't find easier way to append const if arg[1] is set/not set.

Reply:
`$args[1] = ($args[1] ?? 0) | JSON_PRESERVE_ZERO_FRACTION;`

Reply:
Thanks for chiming in @chx . Should we update the code?

Reply:
I think so :)

Reply:
@chx @pySilver Someone wants to take it? ;-)